docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
@nickpan47 ahh, now I remember what you were talking about. Yes, the pipeline feature was going to do something like what you described. We were going to introduce a new "processors" scope that uses the same syntax as this "streams" scope. So to configure a particular processor, configs could be added to 
processors.processorId.*

Note 2 things about this approach:
1. It probably isn't needed until we actually support separate processors. So I wasn't planning on adding it for MVP.
2. Instead of "adding a level above the job" the job now becomes the application and we're adding a level below it. The net result is the same, other than naming (job/application)

I'm not committed to this, but it felt like the least invasive approach from a migration standpoint.

I'll create the JIRA, but will rely on your judgment for priority (MVP or not)